cvtoolkit/datasets/yolo_labels_dataset.py

```diff
@@ -17,6 +18,50 @@ def __init__(self, folder_path: str, image_area: int):
         self._filtered_labels: Dict[str, npt.NDArray] = {}
         self._prepare_labels()
 
+    @classmethod
+    def from_yolo_validation_json(
+        cls, yolo_val_json: str, image_shape: Tuple[int, int]
```

Comment:
can you add a function description similar to the one you added in the PR description  (e.g. the one saved by YOLO validation mode) which also shows what kind of structure is expected? 
This way it can be easily re-used in the future